src/appMain/CMakeLists.txt

```diff
@@ -209,47 +209,31 @@ if (${QT_HMI})
 
   file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/start_hmi.sh
     "#!/bin/sh\n${start_command} ${command_arguments}\n")
-elseif (${WEB_HMI})
-  #file(COPY ${CMAKE_HOME_DIRECTORY}/src/components/HMI/ DESTINATION ${CMAKE_BINARY_DIR}/src/components/HMI/)
-  #file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/hmi_link
-  #  "${CMAKE_BINARY_DIR}/src/components/HMI/index.html")
 endif ()
 
 # Install rules
 install(TARGETS ${PROJECT} DESTINATION bin)
+
 install(
   FILES log4cxx.properties audio.8bit.wav test.txt smartDeviceLink.ini
-    hmi_capabilities.json sdl_preloaded_pt.json
```

Comment:
@LuxoftAKutsan, yes, it should be copied, and it would be. It's done in the next `if` statement.